sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventDataBatch.java

```diff
@@ -22,11 +22,28 @@
 import java.util.Locale;
 import java.util.Objects;
 
-/*
+/**
  * A class for aggregating EventData into a single, size-limited, batch that will be treated as a single message when
  * sent to the Azure Event Hubs service.
+ *
+ * <p><strong>Publish events using an {@link EventDataBatch}.</strong></p>
+ *
+ * Developers can create {@link EventDataBatch batches}, add the events they want in that batch, and publish these
+ * events together. When creating a {@link EventDataBatch batch}, developers can specify a set of {@link BatchOptions
```

Comment:
No need to link to the current class.
```suggestion
 * events together. When creating an {@code EventDataBatch}, developers can specify a set of {@link BatchOptions
```